daemon/daemon_uploads.py

```diff
@@ -85,15 +110,15 @@ async def upload_metadata(session: ClientSession, task: daemon_tasks.Task):
             return msg, None
 
     url = f'{url}{export_data["id"]}/'
-    if "MAINFILE" in upload_set:
+    if "MAINFILE" in upload_set: # quite hidden place to change the status, should be in do_asset_upload explicitly
```

Comment:
Tuhle zmenu verificationStatus pri re-uploadu MAINFILE potrebujeme?
A pokud jo, tak u re-uploadu thumbnailu ji fakt nepotrebujeme?
Co verificationStatus=="uploading" signalizuje?

Potrebuje to vyjasnit, idealne bych to pak strcil do do_asset_upload at je to videt. Anebo strcit tenhle patch i do get_metadata().

Reply:
@PetrDlouhy @vilemduha